packages/fuels-programs/src/script_calls.rs

```diff
@@ -208,18 +208,25 @@ where
             .sum()
     }
 
-    /// Call a script on the node. If `simulate == true`, then the call is done in a
-    /// read-only manner, using a `dry-run`. The [`FuelCallResponse`] struct contains the `main`'s value
-    /// in its `value` field as an actual typed value `D` (if your method returns `bool`,
-    /// it will be a bool, works also for structs thanks to the `abigen!()`).
-    /// The other field of [`FuelCallResponse`], `receipts`, contains the receipts of the transaction.
-    async fn call_or_simulate(&mut self, simulate: bool) -> Result<FuelCallResponse<D>> {
+    /// Returns the script that executes the script call
```

Comment:
```suggestion
    /// Returns the transaction that executes the script call
```
Not sure if I'm correct, but "script...script" sounds off to me